### DISCUSSION TITLE
Image with fixed height and auto width

### DISCUSSION THREAD

**Discussion by GNITOAHC** [vercel/next.js#67938] [Help]
### Summary

I've browse through the document of [NextJs Image](https://nextjs.org/docs/pages/building-your-application/optimizing/images#image-sizing) but I don't see a solution of displaying fix height and auto width. Instead there are ways to display image in fixed width and auto height.

**Comment by romhenri** [comment]
You can set max height and the parent div have a fixed height, so it will be equal.
